lib/index.ts

```diff
@@ -34,6 +36,7 @@ const recommendedRules = {
   'testing-library/await-async-query': 'error',
   'testing-library/await-async-utils': 'error',
   'testing-library/no-await-sync-query': 'error',
+  'testing-library/no-container': 'error',
```

Comment:
Per previous comment, this should added to angular, react and vue presets individually, rather than here.